nikola/data/themes/base-jinja/templates/comments_helper.tmpl

```diff
@@ -1,63 +1,15 @@
 {#  -*- coding: utf-8 -*- #}
 
-{% import 'comments_helper_disqus.tmpl' as disqus with context %}
-{% import 'comments_helper_intensedebate.tmpl' as intensedebate with context %}
-{% import 'comments_helper_muut.tmpl' as muut with context %}
-{% import 'comments_helper_facebook.tmpl' as facebook with context %}
-{% import 'comments_helper_isso.tmpl' as isso with context %}
-{% import 'comments_helper_commento.tmpl' as commento with context %}
-{% import 'comments_helper_utterances.tmpl' as utterances with context %}
+{% import 'comments_helper_%s.tmpl' % comment_system as comment_system_impl with context %}
```

Comment:
This code is different in jinja and mako, which means that an attempt to update themes with `scripts/jinjify.py` will break it. Could you add an override to that script?

Reply:
Done (maybe not the most efficient or concise, but it seems to work - jinjify doesn't change base-jinja anymore now)

Reply:
A `dumb_replacement` would be cleaner than all those regexps IMO — I’ll move the change there.